src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -1014,83 +1031,38 @@ private IMetadata createMetadata() throws FormatException {
    *
    * @param type Bio-Formats pixel type
-   * @return corresponding N5 data type
+   * @return corresponding Zarr data type
    */
-  private DataType getN5Type(int type) {
+  private DataType getZarrType(int type) {
     switch (type) {
       case FormatTools.INT8:
-        return DataType.INT8;
+        return DataType.i1;
       case FormatTools.UINT8:
-        return DataType.UINT8;
+        return DataType.u1;
       case FormatTools.INT16:
-        return DataType.INT16;
+        return DataType.i2;
       case FormatTools.UINT16:
-        return DataType.UINT16;
+        return DataType.u2;
       case FormatTools.INT32:
-        return DataType.INT32;
+        return DataType.i4;
       case FormatTools.UINT32:
-        return DataType.UINT32;
+        return DataType.u4;
       case FormatTools.FLOAT:
-        return DataType.FLOAT32;
+        return DataType.f4;
       case FormatTools.DOUBLE:
-        return DataType.FLOAT64;
+        return DataType.f8;
       default:
         throw new IllegalArgumentException("Unsupported pixel type: "
             + FormatTools.getPixelTypeString(type));
     }
   }
 
-  private DataBlock<?> makeDataBlock(
-    int resolution, int[] size, long[] gridPosition, byte[] tile)
-    throws FormatException
-  {
-    ByteBuffer bb = ByteBuffer.wrap(tile);
-    if (resolution == 0 && isLittleEndian) {
-      bb = bb.order(ByteOrder.LITTLE_ENDIAN);
-    }
-    switch (pixelType) {
-      case FormatTools.INT8:
-      case FormatTools.UINT8: {
-        return new ByteArrayDataBlock(size, gridPosition, tile);
-      }
-      case FormatTools.INT16:
-      case FormatTools.UINT16: {
-        short[] asShort = new short[tile.length / 2];
-        bb.asShortBuffer().get(asShort);
-        return new ShortArrayDataBlock(size, gridPosition, asShort);
-      }
-      case FormatTools.INT32:
-      case FormatTools.UINT32: {
-        int[] asInt = new int[tile.length / 4];
-        bb.asIntBuffer().get(asInt);
-        return new IntArrayDataBlock(size, gridPosition, asInt);
-      }
-      case FormatTools.FLOAT: {
-        float[] asFloat = new float[tile.length / 4];
-        bb.asFloatBuffer().get(asFloat);
-        return new FloatArrayDataBlock(size, gridPosition, asFloat);
-      }
-      case FormatTools.DOUBLE: {
-        double[] asDouble = new double[tile.length / 8];
-        bb.asDoubleBuffer().get(asDouble);
-        return new DoubleArrayDataBlock(size, gridPosition, asDouble);
-      }
-      default:
-        throw new FormatException("Unsupported pixel type: "
-            + FormatTools.getPixelTypeString(pixelType));
-    }
-  }
-
   private void checkOutputPaths() {
-    if (fileType.equals(FileType.zarr) && pyramidName.equals("data.n5")) {
-      pyramidName = "data.zarr";
-    }
-
-    if ((!pyramidName.equals("data.n5") && !pyramidName.equals("data.zarr")) ||
-              !scaleFormatString.equals("%d/%d"))
+    if ((!pyramidName.equals("data.zarr")) ||
```

Comment:
Regarding breaking changes, perhaps also putting the METADATA.ome.xml _within_ the zarr? (Obviously that's not directly related to this PR. But if we want to give them a try, it might make sense to get a few of these larger changes on an integration branch for hammering them out)

Reply:
Could certainly look at bumping the layout and moving the `METADATA.ome.xml` into the Zarr in a separate PR. There are other implications on workflow in doing this which we can discuss there.